Restructure the tutorial into page by page (#491)
nhsuk/nhsuk.service-manual.prototype-kit.docs: app/guides/build-basic-prototype/change-the-service-name.md

---
title: Change the service name
order: 2
---

> You'll normally edit the HTML to make changes to pages, but the service name is in a config file. This is so we can change it in one place to update it on every page in your prototype.

Reviewer: Since this is one of the first steps, we can talk about editing HTML later?

```suggestion
The service name is in a config file so we can update it on every page in your prototype automatically.
```

Author: Good point. Updated in 21c5baf319aefa9e5aae6bb03b43964c5602ac38.

1. Open the `config.js` file in your `app` folder
2. Change `serviceName` from `"Service name goes here"` to `"{{example.name}}"`
3. Press <kbd><kbd>Command</kbd></kbd> + <kbd><kbd>S</kbd></kbd> on Mac or <kbd><kbd>Ctrl</kbd></kbd> + <kbd><kbd>S</kbd></kbd> on Windows to save your change

> You must save every time you make a change to a file. In most code editors, a dot appears in the tab for any file that has unsaved changes.

Reviewer: ```suggestion
You must save every time you make a change to a file. In most code editors, a dot indicator appears in the tab for any file that has unsaved changes.
```

> Normally your changes will automatically show in the browser without refreshing. But for this config change, you need to refresh the page. You should see your service name change on the Start page.

Reviewer: Do you normally sentence case "Start page"?

We lowercase component names for example, but I know GOV don't

Author: I don't think I’ve been very consistent on this either way. Sometimes the capitalisation feels like it helps to make clear you're talking about a specific thing (component/pattern) rather than a general term.

Could also use quote marks or bold (and probably some pages do!).

Author: Done a first pass at making this more consistent in b4657d2836ea4eaa9d26f0b8978630d2ed13aed9
